### PR TITLE
rbiyovskiy/785 codeficator add ordering and city districts to the result of the search request

### PR DESCRIPTION
**Pull request by RBiyovskiy**
Please see details in the description of the issue

### CODE REVIEW

File: OutOfSchool/OutOfSchool.WebApi/Models/Codeficator/AllAddressPartsDto.cs [hunk 1]
```diff
@@ -50,22 +50,22 @@ private void FillAddressPart(CodeficatorDto codeficator, bool isEndPointAddressI
         switch (r2)
```
Comment (valerabad) [inline OutOfSchool/OutOfSchool.WebApi/Models/Codeficator/AllAddressPartsDto.cs:50]: Please remove or rename this variable

File: OutOfSchool/OutOfSchool.Common/Models/CodeficatorAddressDto.cs [hunk 2]
```diff
@@ -20,35 +23,42 @@ public class CodeficatorAddressDto
...
     public double Longitude { get; set; }
 
+    public int Order { get; set; }
+
     public string FullName
     {
         get
         {
-            string addr = CityDistrict;
+            StringBuilder addr = new StringBuilder();
```
Comment (valerabad) [inline OutOfSchool/OutOfSchool.Common/Models/CodeficatorAddressDto.cs:32]: Please set a clear name for the variable

File: OutOfSchool/OutOfSchool.WebApi/Models/AddressDto.cs [hunk 3]
```diff
@@ -30,7 +30,7 @@ public class AddressDto
 
     public double Longitude { get; set; }
 
-    public long CodeficatorId { get; set; }
+    public long CodeficatorId { get; set; } = 31737;
```
Comment (valerabad) [inline OutOfSchool/OutOfSchool.WebApi/Models/AddressDto.cs:33]: I think to move this value to \OutOfSchool.Common\Constants.cs will be better

File: OutOfSchool/OutOfSchool.Common/Models/CodeficatorAddressDto.cs [hunk 4]
```diff
@@ -20,35 +23,42 @@ public class CodeficatorAddressDto
 
     public double Longitude { get; set; }
 
+    public int Order { get; set; }
+
     public string FullName
```
Comment (valerabad) [inline OutOfSchool/OutOfSchool.Common/Models/CodeficatorAddressDto.cs:28]: image

Comment (RBiyovskiy) [reply OutOfSchool/OutOfSchool.Common/Models/CodeficatorAddressDto.cs:28]: I know. This property needed at the front side.

File: OutOfSchool/OutOfSchool.Common/Models/CodeficatorAddressDto.cs (hunk 2 above)
Comment (RBiyovskiy) [reply OutOfSchool/OutOfSchool.Common/Models/CodeficatorAddressDto.cs]: done

File: OutOfSchool/OutOfSchool.WebApi/Models/AddressDto.cs (hunk 3 above)
Comment (RBiyovskiy) [reply OutOfSchool/OutOfSchool.WebApi/Models/AddressDto.cs]: done

**Review by valerabad — APPROVED** [review]

**Review by VadymLevkovskyi — APPROVED** [review]

**Review by DmyMi — APPROVED** [review]

**Pull request merged**